VideoAPI/Testing.Common/Helper/ApiUriFactory.cs

```diff
@@ -82,7 +82,7 @@ public static class TaskEndpoints
         public static class InstantMessageEndpoints
         {
             private const string ApiRoot = "conferences";
-            public static string GetInstantMessageHistory(Guid conferenceId) => $"{ApiRoot}/{conferenceId}/instantmessages";
+            public static string GetInstantMessageHistory(Guid conferenceId, string participantName) => $"{ApiRoot}/{conferenceId}/instantmessages?participantName={participantName}";
```

Comment:
Since the participant name is optional, it might be safer and more coherant to have two properties

GetInstantMessageHistory
GetInstantMessageHistoryFor

Reply:
updated to 2 properties. made participant required.